### PR TITLE
Voltage clamp multi step

### PR DESCRIPTION
**Pull request by alec-hoyland**
I _think_ it works. I've tested it with one of the demos (see `demo_clamp_rk4`) and a couple of toy models. Please try to break it.

### CODE REVIEW

**Comment by sg-s** [comment]
I am truly puzzled that you managed to get it working so easily, I will poke around and see what you did.

**Comment by sg-s** [comment]
I can't find anything wrong with what you've done. Update the relevant bits in the docs in the PR and I'll merge.

**Comment by sg-s** [comment]
you also need to fix this page:

https://xolotl.readthedocs.io/en/master/how-to/voltage-clamp/

**Comment by sg-s** [comment]
Also, there is something very wrong with your text editor if this happens:

Screen Shot 2019-05-08 at 4 28 32 PM

**Comment by alec-hoyland** [comment]
The whitespace thing happens because I have "Remove Trailing Whitespace" set as "true". I have unchecked this, and hopefully it will cease destroying your beautiful trailing spaces.

**Comment by alec-hoyland** [comment]
I am skeptical that my hyperlinks in the docs work.

File: docs/how-to/voltage-clamp.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-## Introduction and Limitations 
+## Introduction and Limitations
 
-Voltage clamping is a technique where you insert an electrode into a cell (or part of a cell), and use a computer to inject whatever current is needed to keep the voltage of that compartment at a desired value. Experimentally, the voltage can be monitored either using a second electrode or by rapidly switching the first electrode between modes where it is used to record the voltage or to inject current. 
+Voltage clamping is a technique where you insert an electrode into a cell (or part of a cell), and use a computer to inject whatever current is needed to keep the voltage of that compartment at a desired value. Experimentally, the voltage can be monitored either using a second electrode or by rapidly switching the first electrode between modes where it is used to record the voltage or to inject current.
 
 xolotl allows you to voltage clamp compartments with the following limitations:
 
 1. Only one compartment can be voltage clamped at a time in any given model.
```
Comment (sg-s) [inline docs/how-to/voltage-clamp.md]: isn't this the line you should change?

**Comment by sg-s** [comment]
You haven't actually changed anything in the the how-to/voltage clamp document?

**Comment by alec-hoyland** [comment]
Yes, I should change that line. Let me play around with it first and see how numerically stable two VCs are first.

**Comment by alec-hoyland** [comment]
Voltage clamping more than one compartment in a network model works well with `solver_order = 0` and `solver_order = 4`. Tested with `demo_stg` model.

File: docs/how-to/voltage-clamp.md (hunk 1 above)
Comment (alec-hoyland) [reply docs/how-to/voltage-clamp.md]: Yes, done.

**Pull request merged**